CMakeLists.txt

```diff
@@ -229,13 +229,6 @@ if(SFML_OS_MACOSX)
         endif()
     endif()
 
-    # only the default x86_64 or arm64 are supported
-    foreach(arch IN LISTS CMAKE_OSX_ARCHITECTURES)
```

Comment:
32 bit iOS devices are a bit more recent than Macs, so want to make sure we don't plan to support them

Reply:
I think 32 bit Macs are around a decade old at this point so I'm not worried about what may happen if someone tries to build SFML on such a machine.